ws-nextjs-app/pages/[service]/send/[id]/MessageBox/ErrorSummaryBox.tsx

```diff
@@ -54,25 +57,28 @@ const ErrorSummaryBox = forwardRef(
         </>
       );
     });
+
+    const contents = isSingleError ? (
+      <Text css={styles.singleItem}>{errorListItems}</Text>
+    ) : (
+      <BulletedList
+        bulletPointColour={WHITE}
+        bulletPointShape="hidden"
+        css={styles.list}
+      >
+        {errorListItems}
+      </BulletedList>
+    );
+
     return (
       <InvalidMessageBox
         id="errorSummaryBox"
         hasArrowStyle={false}
-        messageCode={InvalidMessageCodes.ErrorSummary}
+        messageCode={isAndroid ? null : InvalidMessageCodes.ErrorSummary}
         ref={ref}
         isErrorSummary
       >
-        {isSingleError ? (
-          <Text css={styles.singleItem}>{errorListItems}</Text>
-        ) : (
-          <BulletedList
-            bulletPointColour={WHITE}
-            bulletPointShape="hidden"
-            css={styles.list}
-          >
-            {errorListItems}
-          </BulletedList>
-        )}
+        {isAndroid ? null : contents}
```

Comment:
Apologies I wasn't in the swarm for this, but is an approach to still display the error bullet points but not render them as anchor tags? Visually they'd look the same and would be read out to users, but they wouldn't be able to click on them.